### PR TITLE
remove multiple `workbox-_` DT packages, resolve versions mismatch

### PR DESCRIPTION
**Pull request by NivaldoFarias**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [allowed packages PR](https://github.com/microsoft/DefinitelyTyped-tools/pull/581)
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by peterblazejewicz** [comment]
@NivaldoFarias where are declaration files for workbox-sw? the other seems to ship `.d.ts` files, but not that one?

**Comment by NivaldoFarias** [comment]
> @NivaldoFarias where are declaration files for workbox-sw? the other seems to ship `.d.ts` files, but not that one?

My bad, didn't notice that!! Seems like the package bundles all other workbox packages, but doesn't provide the output declaration files. Will revert the changes to that package,

File: types/customize-cra/tsconfig.json [hunk 1]
```diff
@@ -22,7 +22,8 @@
         },
         "types": [],
         "noEmit": true,
-        "forceConsistentCasingInFileNames": true
+        "forceConsistentCasingInFileNames": true,
+        "esModuleInterop": true
```
Comment (NivaldoFarias) [inline types/customize-cra/tsconfig.json:26]: needed to resolve external dependency modules. see [previous error logs](https://github.com/DefinitelyTyped/DefinitelyTyped/commit/d4a36c4af0b08d724b34c58f7b88a26c74057855/checks?check_suite_id=10019408996#step:6:358)

Comment (peterblazejewicz) [reply types/customize-cra/tsconfig.json:26]: this is becuase of ESM import and 'esModuleInterop' usage in the lib, 'impot * as' woould not impact webpack imports in `.d.ts`. I understand that is not feasable to make this specific import backward compatible (import * as)?

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@NivaldoFarias thanks!

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@NivaldoFarias thanks!

**Comment by NivaldoFarias** [comment]
Ready to merge

**Pull request merged**